src/api/script/import_database.rb

```diff
@@ -0,0 +1,126 @@
+# This script downloads database dump from a remote server and imports it into your local database.
+# It is a script and not a rake task, because downloading from a remote server usually needs
+# a private ssh key. As we're using vagrant in development this would mean that we need to
+# copy the private key into the vagrant box which we avoid with this script.
+require 'yaml'
+require 'fileutils'
+require 'optparse'
+require 'pathname'
+
+TABLES_TO_REMOVE = ['cache_lines', 'project_log_entries']
+params = {}
+
+OptionParser.new do |opts|
+  opts.banner = 'Usage: import_database.rb [options]'
+
+  opts.on('-a', '--all', 'Download and import the latest dump into development database.') do |v|
+    params[:all] = v
+  end
+
+  opts.on('-i', '--import', 'Import the latest dump into development database. You might want to specify the path with --filename.') do |v|
+    params[:import] = v
+  end
+
+  opts.on('-l', '--load', 'Download the latest dump into /db/data directory.') do |v|
+    params[:load] = v
+  end
+
+  opts.on('-p', '--path [PATH]', 'Specify the filename of the database dump. Default is /db/data/obs_production.sql.') do |v|
+    params[:path] = v
+  end
+
+  opts.on('-e', '--environment [PATH]', 'Specify the rails environment. Default is development.') do |v|
+    params[:environment] = v
+  end
+end.parse!
+
+
+def init(params)
+  unless Dir.pwd.include?('open-build-service')
+    abort('Please execute this script in your open-build-service directory')
+  end
+
+# Change in the open-build-service directory
+  while !Dir.pwd.end_with?('open-build-service')
+    Dir.chdir(Pathname(Dir.pwd).dirname)
+  end
+
+  if params.empty?
+    abort('No parameters, use --help')
+  end
+
+  if params[:all] && (params[:import] || params[:load] || params[:path])
+    abort('The --all parameter is not valid in combination with --import, --load or --filename')
+  end
+
+  if params[:load] && params[:filename]
+    abort('The --filename parameter is not valid in combination with --load')
+  end
+end
+
+def load_dump
+  options = YAML.load_file('src/api/config/options.yml')
+  server = options['backup_server']
+  username = options['backup_user']
+  location = options['backup_location']
+  filename = options['backup_filename']
+  port = options['backup_port']
+
+  if !server || !username || !location || !filename
+    abort('Please specify at least backup_server, backup_user, backup_location and backup_filename in your options.yml')
+  end
+
+  puts 'Downloading database backup ...'
+  `scp -P #{port ? port : 22} #{username}@#{server}:#{File.join(location, filename)} src/api/db/data/`
+  puts 'Extracting database backup ...'
+  `bzip2 -d #{File.join('src/api/db/data/', filename)}`
```

Comment:
The data dir is created by default by rails so I think we should use it. Not sure if we want to add the whole dir?